### PR TITLE
[🐴] Reduce amount that message sent date is shown

### PR DESCRIPTION
**Pull request by mozzius**
<table>
  <tr>
    <td>Screenshot 1</td>
    <td>Screenshot 2</td>
  </tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
</table>

### CODE REVIEW

**Review by estrattonbailey — COMMENTED** [review]
Looking so good, posted some questions in Slack to discuss tomorrow 👍

File: src/components/dms/DateDivider.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  yesterday.setDate(today.getDate() - 1)
+  const oneWeekAgo = new Date()
+  oneWeekAgo.setDate(today.getDate() - 7)
+
+  if (localDateString(today) === localDateString(timestamp)) {
+    date = _(msg`Today`)
+  } else if (localDateString(yesterday) === localDateString(timestamp)) {
+    date = _(msg`Yesterday`)
+  } else {
+    if (timestamp > oneWeekAgo) {
```
Comment (estrattonbailey) [inline src/components/dms/DateDivider.tsx:49]: All my dates are coming back with just weekdays, even old chats

File: src/components/dms/DateDivider.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  let date: string
+  const time = timeFormatter.format(new Date(dateStr))
+
+  const timestamp = new Date(dateStr)
+
+  const today = new Date()
+  const yesterday = new Date()
+  yesterday.setDate(today.getDate() - 1)
+  const oneWeekAgo = new Date()
+  oneWeekAgo.setDate(today.getDate() - 7)
```
Comment (estrattonbailey) [inline src/components/dms/DateDivider.tsx:42]: Nit: we've got `date-fns` in here and that has some utils for this if it's any use. Peep `useTimeAgo.test.ts`

**Review by estrattonbailey — COMMENTED** [review]
Looks great to me, def an improvement on what we have now. Thanks for coming back to this!

File: src/components/dms/DateDivider.tsx (hunk 1 above)
Comment (mozzius) [reply src/components/dms/DateDivider.tsx]: Whoops, got the sign the wrong way around

File: src/components/dms/DateDivider.tsx (hunk 2 above)
Comment (mozzius) [reply src/components/dms/DateDivider.tsx]: Ah super helpful, thanks!

**Review by estrattonbailey — APPROVED** [review]

**Pull request merged**